packages/babel-traverse/src/scope/index.js

```diff
@@ -593,7 +593,7 @@ export default class Scope {
       if (node.computed && !this.isPure(node.key, constantsOnly)) return false;
       if (node.kind === "get" || node.kind === "set") return false;
       return true;
-    } else if (t.isClassProperty(node)) {
+    } else if (t.isClassProperty(node) || t.isObjectProperty(node)) {
```

Comment:
Odd that this was checking `ClassProperty` first since that doesn't have a computed property I think?